auth0/src/main/java/com/auth0/android/provider/IdTokenVerifier.java

```diff
@@ -68,14 +68,6 @@ void verify(@NonNull JWT token, @NonNull IdTokenVerificationOptions verifyOption
             throw new TokenValidationException("Issued At (iat) claim must be a number present in the ID token");
         }
 
-        cal.setTime(token.getIssuedAt());
```

Comment:
Having a single mutable calendar object and then setting it to different values betweet tests seems quite an unusual pattern. Is this normal in Java?

Reply:
Think of Calendar as a "dates calculator". It's easier to do _math_ operations on a Calendar than a Date instance. Note though that once the math is done, an instance of Date is obtained from that Calendar for comparison against another one.

Reply:
I get why the Calendar object is being used - it's the re-use for multiple calculations that seems odd.

Reply:
I've always seen it being used like this, at least for when the min JDK is 7. `Calendar` is already a heavy object. There's no harm in re-using it and is better than creating multiple instances.